src/app/work-item/work-item-list/work-item-detail/work-item-link/work-item-link.component.html

```diff
@@ -66,15 +66,26 @@ <h2 class="col-md-12 col-sm-12 col-xs-12 dib">Work Item Links</h2>
                     </div>
                     <div class="col-md-12 col-sm-12 col-xs-12">
                         <div class="col-md-9 col-sm-9 col-xs-9">
-                            <ng2-completer (selected)="onSelectItem($event)" 
-                              [dataService]="searchWorkItem" 
-                              [minSearchLength]="2" 
-                              [placeholder]="'search for work items'"
-                              [textSearching]="'Please wait...'">
-                            </ng2-completer> 
+                            <input id="workitem-link-search" #searchBox
```

Comment:
That is an angular ID to grab the element in component using ViewChild or ViewChildren.